tests/wallet/nft_wallet/test_nft_offers.py

```diff
@@ -912,288 +905,3 @@ async def test_nft_offer_nft_for_nft(
 
     assert await nft_wallet_maker.get_nft_count() == 1
     assert await nft_wallet_taker.get_nft_count() == 1
-
-
-@pytest.mark.limit_consensus_modes(allowed=[ConsensusMode.PLAIN], reason="save time")
-@pytest.mark.parametrize("trusted", [True, False])
-@pytest.mark.parametrize("reuse_puzhash", [True, False])
-@pytest.mark.anyio
-async def test_nft_offer_nft0_and_xch_for_cat(
```

Comment:
why was this test removed?

Reply:
Good catch, the answer is "on accident" :)